crates/router/src/services/api/generic_link_response.rs

```diff
@@ -149,7 +122,7 @@ pub fn build_pm_collect_link_html(
         ),
     );
 
-    tera.render("payment_method_collect_link", &context)
+    tera.render("html_template", &context)
         .change_context(errors::ApiErrorResponse::InternalServerError)
         .attach_printable("Failed to render payment method collect link's HTML template")
 }
```

Comment:
we can also, unify status page for payout_link and pm_collect link

Reply:
for later!